ci/cron/daily-compat.yml

```diff
@@ -24,6 +24,7 @@ schedules:
 
 jobs:
   - job: compatibility_ts_libs
+    condition: false
```

Comment:
I assume you don’t want to merge all of those?

Reply:
Indeed. They're part of a separate commit that I'll remove before merging.